eng/packages/http-client-csharp-mgmt/generator/Azure.Generator.Management/src/ManagementOutputLibrary.cs

```diff
@@ -36,6 +36,12 @@ public class ManagementOutputLibrary : AzureOutputLibrary
         private WirePathAttributeDefinition? _wirePathAttributeProvider;
         internal TypeProvider WirePathAttributeDefinition => _wirePathAttributeProvider ??= new WirePathAttributeDefinition();
 
+        private CSharpType? _modelReaderWriterContextType;
+        internal CSharpType ModelReaderWriterContextType => _modelReaderWriterContextType ??= GetModelReaderWriterContextType();
+
+        private string? _modelReaderWriterContextTypeName;
+        internal string ModelReaderWriterContextTypeName => _modelReaderWriterContextTypeName ??= GetModelReaderWriterContextTypeName();
```

Comment:
I think you did not get my point.
So here, we should only keep this `ModelReaderWriterContextType` property and remove the other.
It should get the type in this way:
```
internal CSharpType ModelReaderWriterContextType => _modelReaderWriterContextType ??= new ModelReaderWriterContextDefinition().Type;
```